genesis/process/metaGenesisBlockCreator.go

```diff
@@ -167,6 +165,23 @@ func createMetaGenesisAfterHardFork(
 	return metaHdr, make([][]byte, 0), nil
 }
 
+func createArgsMetaBlockCreatorAfterHardFork(
```

Comment:
This PR will cause a lot of conflicts on soft-fork feature branch

Reply:
my bad :)